libqtile/widget/base.py

```diff
@@ -233,7 +234,7 @@ def add_callbacks(self, defaults):
 
     def button_press(self, x, y, button):
         name = 'Button{0}'.format(button)
-        if name in self.mouse_callbacks:
+        if self.mouse_callbacks and name in self.mouse_callbacks:
```

Comment:
nor this, actually (continuation of previous comment)

Reply:
As in putting:

`if self.mouse_callbacks is not None:`

In the add_callbacks func of the base widget so none of its code runs?

Reply:
I think we still need this one because `name in self.mouse_callbacks:`
throws `TypeError: argument of type 'NoneType' is not iterable`.
without it.

Reply:
Yeah right, this one is still needed. For my defense it was like 4 AM when I wrote the comment, sorry! :D

Reply:
no prob :)